test.sh

```diff
@@ -16,7 +16,10 @@
 
 source base.sh
 
-#Test for codeworld-compiler
-run codeworld-compiler cabal configure --enable-tests
-run codeworld-compiler cabal build 
-run codeworld-compiler cabal test
+DirList="codeworld-compiler"
```

Comment:
If you wanted to be even fancier, you could try:

    for i in $(grep -l -i Test-Suite */*.cabal); do
        TARGET=$(basename $i)
        run $TARGET cabal configure --enable-tests
        run $TARGET cabal build
        run $TARGET cabal test
    done

What do you think?